src/builders/indexable-post-builder.php

```diff
@@ -108,29 +118,30 @@ public function build( $post_id, $indexable ) {
 		$indexable->permalink       = $this->get_permalink( $post->post_type, $post_id );
 
 		$indexable->primary_focus_keyword_score = $this->get_keyword_score(
-			$this->get_meta_value( $post_id, 'focuskw' ),
-			(int) $this->get_meta_value( $post_id, 'linkdex' )
+			$this->meta->get_value( 'focuskw', $post_id ),
+			(int) $this->meta->get_value( 'linkdex', $post_id )
 		);
 
-		$indexable->readability_score = (int) $this->get_meta_value( $post_id, 'content_score' );
+		$indexable->readability_score = (int) $this->meta->get_value( 'content_score', $post_id );
 
-		$indexable->is_cornerstone    = ( $this->get_meta_value( $post_id, 'is_cornerstone' ) === '1' );
+		$indexable->is_cornerstone    = ( $this->meta->get_value( 'is_cornerstone', $post_id ) === '1' );
 		$indexable->is_robots_noindex = $this->get_robots_noindex(
-			$this->get_meta_value( $post_id, 'meta-robots-noindex' )
+			(int) $this->meta->get_value( 'meta-robots-noindex', $post_id )
 		);
 
 		// Set additional meta-robots values.
-		$indexable->is_robots_nofollow = ( $this->get_meta_value( $post_id, 'meta-robots-nofollow' ) === '1' );
-		$noindex_advanced              = $this->get_meta_value( $post_id, 'meta-robots-adv' );
+		$indexable->is_robots_nofollow = ( $this->meta->get_value( 'meta-robots-nofollow', $post_id ) === '1' );
+		$noindex_advanced              = $this->meta->get_value( 'meta-robots-adv', $post_id );
 		$meta_robots                   = \explode( ',', $noindex_advanced );
+
 		foreach ( $this->get_robots_options() as $meta_robots_option ) {
```

Comment:
Not directly related to this PR: why does the `get_robots_options()` exist ? It only returns a constant array, why is this not a constant ? (or even a property if you want to keep it ` private`)

Reply:
Good question. Looks like there is no good reason for this method to exist. I'd love to move it to a private const once we drop PHP 5.6. Moving it to a private property doesn't feel completely right to me because it hints to being mutable, which it isn't.

Reply:
Unfortunately constant visibility will have to wait as it is a PHP 7.1+ feature and yeah, I'm totally fine with leaving this until support for PHP < 7.1 has been dropped.